java-snapshot-testing-core/src/main/java/au/com/origin/snapshots/comparators/CompareUtils.java

```diff
@@ -0,0 +1,14 @@
+package au.com.origin.snapshots.comparators;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class CompareUtils {
```

Comment:
This should probably go entirely because we should aim to eliminate the coupling between comparing and diff reporting.

Reply:
Got it.